internal/command/jsonformat/plan_test.go

```diff
@@ -7849,12 +7849,12 @@ func runTestCases(t *testing.T, testCases map[string]testCase) {
 
 			beforeDynamicValue, err := plans.NewDynamicValue(beforeVal, ty)
 			if err != nil {
-				t.Fatalf("failed to create dynamic before value: " + err.Error())
+				t.Fatalf("failed to create dynamic before value: %s", err.Error())
```

Comment:
If you're changing these to formatted strings, `%s` calls `Error()` implicitly.

Reply:
Ah, good point! I'll remove the `Error()`

Reply:
Addressed in 07fe9dbe929f775819839790ef4ac88d515b3cf4